releasenotes/notes/higher-order-spsa-f2eddb44b00b82e0.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Add the [2-SPSA](https://ieeexplore.ieee.org/document/657661) and
+    [QN-SPSA](https://arxiv.org/abs/2103.09232) optimizers.
```

Comment:
Can we have a reference to the actual module being added here. Right now you're talking all about the algorithm but not actually showing where it is or how to use it. This is only half of a release note.

Reply:
I added a reference to the class, whose docstring contains an example 🙂